### PR TITLE
docs(aps): update docs apple silicon

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: bare-metal/apple-silicon/quickstart.mdx [hunk 1]
```diff
@@ -24,63 +25,63 @@ Apple silicon is designed for developing, building, testing, and signing applica
...
+  The default user to connect to your Mac mini is `m1`. The SSH command to connect is `ssh @<your_mac_mini__ip>`.
 </Message>
 
-## How to delete a Mac mini M1
+## How to delete a Mac mini
 
-1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac mini M1s displays.
-2. Click <Icon name="more" /> next to the Mac mini M1 you want to delete, and select **Delete** from the drop-down menu. A pop-up asks you to confirm the action.
-3. Type **DELETE** and then click **Delete Mac mini M1**
+1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac mini s displays.
```
Comment (jsudraud) [inline bare-metal/apple-silicon/quickstart.mdx:80]:
```suggestion
1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac minis displays.
```

File: bare-metal/apple-silicon/quickstart.mdx [hunk 2]
```diff
@@ -24,63 +25,63 @@ Apple silicon is designed for developing, building, testing, and signing applica
...
+2. Click <Icon name="more" /> next to the Mac mini you want to delete, and select **Delete** from the drop-down menu. A pop-up asks you to confirm the action.
+3. Type **DELETE** and then click **Delete Mac mini **
 
-You are returned to the list of your Mac mini M1s, where the machine you deleted no longer appears.
+You are returned to the list of your Mac mini s, where the machine you deleted no longer appears.
```
Comment (jsudraud) [inline bare-metal/apple-silicon/quickstart.mdx:84]:
```suggestion
You are returned to the list of your Mac minis, where the machine you deleted no longer appears.
```

File: bare-metal/apple-silicon/quickstart.mdx [hunk 3]
```diff
@@ -24,63 +25,63 @@ Apple silicon is designed for developing, building, testing, and signing applica
...
     <Lightbox src="scaleway-m1-remote-desktop.webp" alt="" />
 
 ### How to connect via SSH
 
-You can also connect directly to the terminal of your Mac mini M1 using the SSH protocol and your [SSH key](/console/account/concepts/#ssh-key). 
+You can also connect directly to the terminal of your Mac mini using the SSH protocol and your [SSH key](/console/account/concepts/#ssh-key). 
 
-1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac mini M1s displays.
-2. Click the Mac mini M1 you want to connect to. Your Mac mini M1 **Overview** displays. You can find your SSH connection information here, under **SSH command**.
+1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac mini s displays.
```
Comment (jsudraud) [inline bare-metal/apple-silicon/quickstart.mdx:68]:
```suggestion
1. Click **Apple silicon** in the **Bare Metal** section of the side menu. A list of your Mac minis displays.
```

**Review by jsudraud — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Comment by scwgoire** [comment]
Looks nice, 2 remarks:
- what's the value in telling the macs are mac minis? I don't think it hurt but it doesn't seem to help either
- please hold the MR until we resolve the ongoing discussions about the upcoming products (cf Slack)

**Pull request merged**